p.cabal

```diff
@@ -14,8 +14,9 @@ description:           A set of standard preludes for Ambiata projects. \
 
 library
   build-depends:
-                       base                            >= 4.6        && < 5
-                     , semigroups                      == 0.16.*
+                         base                        >= 4.6        && < 5
+                       , either                      >= 4.4.1      && < 5
```

Comment:
Right, I made a mistake in thinking that starting with 4.3 would prevent me to use 4.4.1 in my project.